src/main/java/org/opensearch/geospatial/action/upload/geojson/UploadGeoJSONResponse.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial.action.upload.geojson;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.opensearch.action.ActionResponse;
+import org.opensearch.action.bulk.BulkItemResponse;
+import org.opensearch.action.bulk.BulkResponse;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.common.xcontent.ToXContentObject;
+import org.opensearch.common.xcontent.XContentBuilder;
+
+//UploadGeoJSONResponse represents UploadGeoJSONRequest's Response
+public class UploadGeoJSONResponse extends ActionResponse implements ToXContentObject {
+    private static final String ERRORS = "errors";
+    private static final String FAILURE = "failure";
+    private static final String ID = "id";
+    private static final String MESSAGES = "messages";
+    private static final String REASON = "reason";
+    private static final String SUCCESS = "success";
+    private static final String TOOK_IN_MILLI_SECOND = "took_in_ms";
+    private static final String TOTAL = "total";
+    private static final int NO_FAILURE = 0;
+    private final BulkResponse response;
+
+    public UploadGeoJSONResponse(BulkResponse response) {
+        this.response = response;
+    }
+
+    public UploadGeoJSONResponse(StreamInput in) throws IOException {
+        super(in);
+        this.response = new BulkResponse(in);
+    }
+
+    @Override
+    public void writeTo(StreamOutput streamOutput) throws IOException {
+        this.response.writeTo(streamOutput);
+    }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        /*
+        If BulkResponse has no failures:
```

Comment:
Why cant we just return the BulkResponse?

Reply:
Good point. [BulkResponse](https://opensearch.org/docs/latest/opensearch/rest-api/document-apis/bulk/#response) exposes too many fields that are not relevant from this API perspective. eg: shard id/seq_no, etc. WDYT?

Reply:
I think that makes sense. Do we want to include failure status in messages?

Reply:
you mean error code?